packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { TableContainer } from './TableContainer';
+import { TableHeaderCell } from './TableHeaderCell';
+import { TableHeader } from './TableHeader';
+import { TableRow } from './TableRow';
+import { TableRowCell } from './TableRowCell';
+
+const EXAMPLE_TOKENS = [
+    { name: 'USDT', balance: '10', price: '$1,00' },
+    { name: 'USDC', balance: '100', price: '$1.00' },
+    { name: 'MANA', balance: '20', price: '$2.15' },
+];
+
+const meta: Meta = {
+    title: 'Table',
+} as Meta;
+export default meta;
+
+export const Table: StoryObj = {
```

Comment:
Could you please add also `margin` from `frameProps`? 🙏 


Plese copy implementation from here

```
packages/components/src/components/Divider/Divider.tsx
```

and storybook's args/argTypes from here:
```
packages/components/src/components/Divider/Divider.stories.tsx